semcore/popper/src/Popper.jsx

```diff
@@ -50,6 +50,13 @@ const useUpdatePopperEveryFrame = (popperRef) => {
   return handleAnimationFrame;
 };
 
+let lastMouseMove = 0;
+if (canUseDOM()) {
+  document.addEventListener('mousemove', () => {
+    lastMouseMove = Date.now();
+  });
+}
```

Comment:
are you sure about place for this code? Maybe it's better to put it inside Popper class?

Reply:
Yes, I quite sure because we can have multiple poppers on page but lastMouseMove information maybe safely shared